### ISSUE TITLE
Walls switching graphics?

### ISSUE THREAD

**Issue by squiblez** [ethanmoffat/EndlessClient#38]
I started encountering sprites/textures that seem to be cycling like animations, and the glitch I had encountered before on the same maps(Map 12, and the castle interior).

At first I thought it could have been some of the changes I've made to the code(experimenting with features I've always been curious about having in EO, which DID include modifying the way certain things are rendered). But after trying the unmodified code I experienced the same issue. After a few tries the map loaded without throwing any exceptions, and that's where things got trippy.

![screenshot1](https://cloud.githubusercontent.com/assets/14168449/9835779/52084330-59c9-11e5-9f8a-97d0045d4d38.jpg)
![screenshot2](https://cloud.githubusercontent.com/assets/14168449/9835780/5573ef7e-59c9-11e5-9266-682fd2d6d2f7.jpg)

But on a side note, after spending a few days getting a feel for it, this is absolutely amazing. The code is smooth and consistent, easy to manage and expand. You have talent.

**Comment by ethanmoffat** [comment]
> But on a side note, after spending a few days getting a feel for it, this is absolutely amazing. The code is smooth and consistent, easy to manage and expand. You have talent.

Thank you! I'm glad it works for you :) honestly I made quite a few bad design decisions early on that could have been avoided with better planning, but I like to think that overall the code isn't completely terrible.

Anyway, this error occurs as a result of how big I expected animated wall tiles to be. My assumption was anything with width > 32px should be animated. I've since adjusted this to included edge cases, raising the lower limit to 68px so these wall tiles don't animate. AFAIK it hasn't broken any existing wall animations.

This fix should be submitted in the next few hours.
